components/EmotionList.js

```diff
@@ -5,7 +5,7 @@ import EmotionCard from "./EmotionCard";
 export default function EmotionList({ objects }) {
   return (
     <StyledEmotionList>
-      {objects.length === 0 ? ( // Überprüfe, ob keine Einträge vorhanden sind
+      {objects.length === 0 ? (
```

Comment:
Instead of a ternary operator, I would recommend the short form with `&&`, i.e. `{objects.length === 0 && (...)}` and then the other case below. But that's a matter of taste and of course you can do it the way you did. I find the short form a bit clearer than the ternary operators, especially when the code becomes more complex.